### ISSUE TITLE
TopCalc

### ISSUE THREAD

**Issue by Macmada** [olifolkerd/tabulator#1307]
Hello to you,
When setting up the TopCalc on a table with a lot of column, some calculations are well displayed and from a number of columns, the display becomes grayed out.

The problem does not exist with the bottomCalc.

Is it possible to see please?

thank you

TopCalc : 
![topcalc](https://user-images.githubusercontent.com/41945143/43446596-9605fafa-94a9-11e8-95e8-0015cff6ed38.PNG)
BottomCalc :
![bottomcalc](https://user-images.githubusercontent.com/41945143/43446595-95ee3fd2-94a9-11e8-8175-5e487656ae9e.PNG)

**Comment by olifolkerd** [comment]
Hey @Macmada 

Sorry to hear you are having issues,

Could you confirm which version of Tabulator you are using, which browser you are on and what your table constructor is?

Can you replicate the issue in a code pen?

Cheers

Oli :)

**Comment by Macmada** [comment]
Hello to you,

Thank you for your return and especially for TABULATOR which is a very good tool.

Here are the answers to your questions :
TABULATOR Version: 3.5.3
CHROME version: 67.0

CODE TABULATOR : 
```
$("#form_secteur").change(function () {
		secteur = $('input:radio[name="secteur"]:checked').val();
		time = $('input:radio[name="time"]:checked').val();
		nb_mois = $('input:radio[name="nbmois"]:checked').val();
		$("#table_equi_sec").show();
		url_sec = 'db/db_irm_cp_eff.php?secteur=' + secteur
		$("#table_equi_sec").tabulator({
			height: "100%",
			layout: "fitData",
			virtualDom: false,
			renderComplete: function () {
				color5()
			},
		})
		base_col = [{ title: "Mat", field: "matricule", frozen: true }, { title: "Poste", field: "poste", frozen: true, headerFilter:true }, { title: "Dept", field: "dept", frozen: true, headerFilter: true, headerFilterFunc: filtre_dept }, { title: "Nom", field: "nom_x", frozen: true, headerFilter : true }, { title: "Prénom", field: "prenom_x", frozen: true }, { title: "Amp", field: "amp", frozen: true, },{title:"Skill",field:"skill",frozen:true,headerFilter: true}, { title: "CP", field: "solde_cp", frozen: true, width:105,headerFilter:minMaxFilterEditor, headerFilterFunc:minMaxFilterFunction},]

		col_date = [
{title:"2018-08-06",field:"2018-08-06", bottomCalc:presence},{title:"2018-08-07",field:"2018-08-07", bottomCalc:presence},{title:"2018-08-08",field:"2018-08-08", bottomCalc:presence},{title:"2018-08-09",field:"2018-08-09", bottomCalc:presence},{title:"2018-08-10",field:"2018-08-10", bottomCalc:presence},{title:"2018-08-11",field:"2018-08-11", bottomCalc:presence},{title:"2018-08-12",field:"2018-08-12", bottomCalc:presence},{title:"2018-08-13",field:"2018-08-13", bottomCalc:presence},{title:"2018-08-14",field:"2018-08-14", bottomCalc:presence},{title:"2018-08-15",field:"2018-08-15", bottomCalc:presence},{title:"2018-08-16",field:"2018-08-16", bottomCalc:presence},{title:"2018-08-17",field:"2018-08-17", bottomCalc:presence},{title:"2018-08-18",field:"2018-08-18", bottomCalc:presence},{title:"2018-08-19",field:"2018-08-19", bottomCalc:presence},{title:"2018-08-20",field:"2018-08-20", bottomCalc:presence},{title:"2018-08-21",field:"2018-08-21", bottomCalc:presence},{title:"2018-08-22",field:"2018-08-22", bottomCalc:presence},{title:"2018-08-23",field:"2018-08-23", bottomCalc:presence},{title:"2018-08-24",field:"2018-08-24", bottomCalc:presence},{title:"2018-08-25",field:"2018-08-25", bottomCalc:presence},{title:"2018-08-26",field:"2018-08-26", bottomCalc:presence},{title:"2018-08-27",field:"2018-08-27", bottomCalc:presence},{title:"2018-08-28",field:"2018-08-28", bottomCalc:presence},{title:"2018-08-29",field:"2018-08-29", bottomCalc:presence},{title:"2018-08-30",field:"2018-08-30", bottomCalc:presence},{title:"2018-08-31",field:"2018-08-31", bottomCalc:presence},{title:"2018-09-01",field:"2018-09-01", bottomCalc:presence},{title:"2018-09-02",field:"2018-09-02", bottomCalc:presence},{title:"2018-09-03",field:"2018-09-03", bottomCalc:presence},{title:"2018-09-04",field:"2018-09-04", bottomCalc:presence},{title:"2018-09-05",field:"2018-09-05", bottomCalc:presence},{title:"2018-09-06",field:"2018-09-06", bottomCalc:presence},{title:"2018-09-07",field:"2018-09-07", bottomCalc:presence},{title:"2018-09-08",field:"2018-09-08", bottomCalc:presence},{title:"2018-09-09",field:"2018-09-09", bottomCalc:presence},{title:"2018-09-10",field:"2018-09-10", bottomCalc:presence},{title:"2018-09-11",field:"2018-09-11", bottomCalc:presence},{title:"2018-09-12",field:"2018-09-12", bottomCalc:presence},{title:"2018-09-13",field:"2018-09-13", bottomCalc:presence},{title:"2018-09-14",field:"2018-09-14", bottomCalc:presence},{title:"2018-09-15",field:"2018-09-15", bottomCalc:presence},{title:"2018-09-16",field:"2018-09-16", bottomCalc:presence},{title:"2018-09-17",field:"2018-09-17", bottomCalc:presence},{title:"2018-09-18",field:"2018-09-18", bottomCalc:presence},{title:"2018-09-19",field:"2018-09-19", bottomCalc:presence},{title:"2018-09-20",field:"2018-09-20", bottomCalc:presence},{title:"2018-09-21",field:"2018-09-21", bottomCalc:presence},{title:"2018-09-22",field:"2018-09-22", bottomCalc:presence},{title:"2018-09-23",field:"2018-09-23", bottomCalc:presence},{title:"2018-09-24",field:"2018-09-24", bottomCalc:presence},{title:"2018-09-25",field:"2018-09-25", bottomCalc:presence},{title:"2018-09-26",field:"2018-09-26", bottomCalc:presence},{title:"2018-09-27",field:"2018-09-27", bottomCalc:presence},{title:"2018-09-28",field:"2018-09-28", bottomCalc:presence},{title:"2018-09-29",field:"2018-09-29", bottomCalc:presence},{title:"2018-09-30",field:"2018-09-30", bottomCalc:presence},{title:"2018-10-01",field:"2018-10-01", bottomCalc:presence},{title:"2018-10-02",field:"2018-10-02", bottomCalc:presence},{title:"2018-10-03",field:"2018-10-03", bottomCalc:presence},{title:"2018-10-04",field:"2018-10-04", bottomCalc:presence},{title:"2018-10-05",field:"2018-10-05", bottomCalc:presence},{title:"2018-10-06",field:"2018-10-06", bottomCalc:presence},{title:"2018-10-07",field:"2018-10-07", bottomCalc:presence},{title:"2018-10-08",field:"2018-10-08", bottomCalc:presence},{title:"2018-10-09",field:"2018-10-09", bottomCalc:presence},{title:"2018-10-10",field:"2018-10-10", bottomCalc:presence},{title:"2018-10-11",field:"2018-10-11", bottomCalc:presence},{title:"2018-10-12",field:"2018-10-12", bottomCalc:presence},{title:"2018-10-13",field:"2018-10-13", bottomCalc:presence},{title:"2018-10-14",field:"2018-10-14", bottomCalc:presence},{title:"2018-10-15",field:"2018-10-15", bottomCalc:presence},{title:"2018-10-16",field:"2018-10-16", bottomCalc:presence},{title:"2018-10-17",field:"2018-10-17", bottomCalc:presence},{title:"2018-10-18",field:"2018-10-18", bottomCalc:presence},{title:"2018-10-19",field:"2018-10-19", bottomCalc:presence},{title:"2018-10-20",field:"2018-10-20", bottomCalc:presence},{title:"2018-10-21",field:"2018-10-21", bottomCalc:presence},{title:"2018-10-22",field:"2018-10-22", bottomCalc:presence},{title:"2018-10-23",field:"2018-10-23", bottomCalc:presence},{title:"2018-10-24",field:"2018-10-24", bottomCalc:presence},{title:"2018-10-25",field:"2018-10-25", bottomCalc:presence},{title:"2018-10-26",field:"2018-10-26", bottomCalc:presence},{title:"2018-10-27",field:"2018-10-27", bottomCalc:presence},{title:"2018-10-28",field:"2018-10-28", bottomCalc:presence},{title:"2018-10-29",field:"2018-10-29", bottomCalc:presence},{title:"2018-10-30",field:"2018-10-30", bottomCalc:presence},{title:"2018-10-31",field:"2018-10-31", bottomCalc:presence},{title:"2018-11-01",field:"2018-11-01", bottomCalc:presence},{title:"2018-11-02",field:"2018-11-02", bottomCalc:presence},{title:"2018-11-03",field:"2018-11-03", bottomCalc:presence},{title:"2018-11-04",field:"2018-11-04", bottomCalc:presence},{title:"2018-11-05",field:"2018-11-05", bottomCalc:presence},{title:"2018-11-06",field:"2018-11-06", bottomCalc:presence},{title:"2018-11-07",field:"2018-11-07", bottomCalc:presence},{title:"2018-11-08",field:"2018-11-08", bottomCalc:presence},{title:"2018-11-09",field:"2018-11-09", bottomCalc:presence},{title:"2018-11-10",field:"2018-11-10", bottomCalc:presence},{title:"2018-11-11",field:"2018-11-11", bottomCalc:presence},{title:"2018-11-12",field:"2018-11-12", bottomCalc:presence},{title:"2018-11-13",field:"2018-11-13", bottomCalc:presence},{title:"2018-11-14",field:"2018-11-14", bottomCalc:presence},{title:"2018-11-15",field:"2018-11-15", bottomCalc:presence},{title:"2018-11-16",field:"2018-11-16", bottomCalc:presence},{title:"2018-11-17",field:"2018-11-17", bottomCalc:presence},{title:"2018-11-18",field:"2018-11-18", bottomCalc:presence},{title:"2018-11-19",field:"2018-11-19", bottomCalc:presence},{title:"2018-11-20",field:"2018-11-20", bottomCalc:presence},{title:"2018-11-21",field:"2018-11-21", bottomCalc:presence},{title:"2018-11-22",field:"2018-11-22", bottomCalc:presence},{title:"2018-11-23",field:"2018-11-23", bottomCalc:presence},{title:"2018-11-24",field:"2018-11-24", bottomCalc:presence},{title:"2018-11-25",field:"2018-11-25", bottomCalc:presence},{title:"2018-11-26",field:"2018-11-26", bottomCalc:presence},{title:"2018-11-27",field:"2018-11-27", bottomCalc:presence},{title:"2018-11-28",field:"2018-11-28", bottomCalc:presence},{title:"2018-11-29",field:"2018-11-29", bottomCalc:presence},{title:"2018-11-30",field:"2018-11-30", bottomCalc:presence},{title:"2018-12-01",field:"2018-12-01", bottomCalc:presence},{title:"2018-12-02",field:"2018-12-02", bottomCalc:presence},{title:"2018-12-03",field:"2018-12-03", bottomCalc:presence},{title:"2018-12-04",field:"2018-12-04", bottomCalc:presence},{title:"2018-12-05",field:"2018-12-05", bottomCalc:presence},{title:"2018-12-06",field:"2018-12-06", bottomCalc:presence},];
		col_final = [];

		for (var i = 0; i < base_col.length; i++) { col_final.push(base_col[i]) }


		if (nb_mois == 1) {
			for (var i = 0; i < 32; i++) { col_final.push(col_date[i]) }
		}
		else if (nb_mois == 2) {
			for (var i = 0; i < 64; i++) { col_final.push(col_date[i]) }
		}
		else if (nb_mois == 3) {
			for (var i = 0; i < 96; i++) { col_final.push(col_date[i]) }
		}
		else if (nb_mois == 4) {
			for (var i = 0; i < 123; i++) { col_final.push(col_date[i]) }
		}

		$("#table_equi_sec").tabulator("setColumns", col_final)
		$("#table_equi_sec").tabulator("setData", url_sec);

	});
```

I try tomorrow to make you a CODEPEN

thanks again

**Comment by olifolkerd** [comment]
Hey @Macmada 

Thanks for getting in touch, a fix for this has just been pushed to the 4.0 branch which should be released later on today.

Cheers

Oli :)
